.travis.yml

```diff
@@ -4,7 +4,8 @@ sudo: false
 
 env:
   global:
-    - CONDA_DEPENDENCIES="pytest numpy astropy"
+    - CONDA_DEPENDENCIES="pytest pytest-cov numpy astropy"
```

Comment:
weird, pytest-cov install should be covered by ci-helpers. Can you open an issue there as a reminder pointing to the failing travis?

Reply:
I'm not using ``$SETUP_CMD`` and ``--coverage`` though, so this is fine

Reply:
oh, yes indeed, I haven't noticed the missing `$SETUP_CMD`. The standard `--cov packagename ` should work though for a while now.